src/cluda_opencl.h.c

```diff
@@ -0,0 +1,107 @@
+static const char cluda_opencl_h[] = {
```

Comment:
This is to embed the source code in cluda_opencl.h in the binary so that we can use it for kernels.